pkgs/build-support/rust/hooks/maturin-build-hook.sh

```diff
@@ -12,6 +12,11 @@ maturinBuildHook() {
         pushd "${buildAndTestSubdir}"
     fi
 
+    # This is a huge hack, but it's the least invasive way
+    # to get the required interpreter name for maturin.
+    local interpreter_path="$(command -v python || command -v pypy3)"
```

Comment:
maybe probe for `python3` instead of `python`? Just to make sure in case there is a python 2 in the env.